README.md

```diff
@@ -27,21 +38,29 @@ resource "aws_iam_role_policy_attachment" "codebuild_s3" {
-| image         | alpine               | Docker image used as environment                                                                                               |
-| instance_size | BUILD_GENERAL1_SMALL | Instance size for job.  Possible values are: ```BUILD_GENERAL1_SMALL``` ```BUILD_GENERAL1_MEDIUM``` ```BUILD_GENERAL1_LARGE``` |
-| buildspec     | ""                   | Optional buildspec declaration to use for building the project                                                                 |
+| Name            | Default              | Description                                                                                                                                          |
+|:---------------:|:--------------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------:|
+| namespace       | global               | Namespace                                                                                                                                            |
+| stage           | default              | Stage                                                                                                                                                |
+| name            | codebuild            | Name                                                                                                                                                 |
+| image           | ""                   | Docker image used for build environment, _e.g._ `aws/codebuild/docker:1.12.1` or `aws/codebuild/eb-nodejs-6.10.0-amazonlinux-64:4.0.0`               |
+| instance_size   | BUILD_GENERAL1_SMALL | CodeBuild instance size.  Possible values are: ```BUILD_GENERAL1_SMALL``` ```BUILD_GENERAL1_MEDIUM``` ```BUILD_GENERAL1_LARGE```                     |
+| buildspec       | ""                   | (Optional) `buildspec` declaration to use for building the project                                                                                       |
+| privileged_mode | ""                   | (Optional) If set to true, enables running the Docker daemon inside a Docker container on the `CodeBuild` instance. Used when building Docker images |
+| aws_region      | ""                   | (Optional) AWS Region, _e.g._ `us-east-1`. Used as `CodeBuild` ENV variable when building Docker images                                              |
+| aws_account_id  | ""                   | (Optional) AWS Account ID. Used as `CodeBuild` ENV variable when building Docker images                                                              |
+| image_repo_name | ""                   | (Optional) ECR repository name to store Docker images. Used as `CodeBuild` ENV variable when building Docker images                                  |
```

Comment:
Ok, I think we should disambiguate between the `image` used for the build environment and the `image_repo_name` which where stuff gets pushed. Maybe we should rename `image` to `build_image`

Reply:
(like we have `buildspec`)

Reply:
Renamed